cfgov/v1/util/ref.py

```diff
@@ -133,6 +133,24 @@
     ('ht', 'Haitian Creole'),
 ]
 
+snippet_types = [
```

Comment:
I'm curious, do we even need to hardcode these types at all? Wagtail has a built-in [`wagatilsnippets.models.get_snippets`](https://github.com/wagtail/wagtail/blob/v1.7/wagtail/wagtailsnippets/models.py#L11) method that returns a list of all registered snippet models (those registered with `@register_snippet`).

We could set the [`verbose_name_plural`](https://docs.djangoproject.com/en/1.8/ref/models/options/#verbose-name-plural) on `class Meta` our existing snippet models (or just use the default, since it'll work in this case), and then we don't need this hardcoded list at all.

Similarly, for the fields, those could be defined on the models themselves (as a class member or a method).